modules/ROOT/pages/push/proc_sending-a-push-notification.adoc

```diff
@@ -0,0 +1,40 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: proc_sending-a-push-notification
+
+[id='{context}_sending-a-push-notification']
+
+= Sending a Push Notification
+
+The {push-ui} allows you to send {push-notification}s to devices.
+
+[discrete]
+== Prerequisites
+
+* Access to the {push-ui}.
+* An application with at least one binding or variant.
+
+[discrete]
+== Procedure
+
+. Open the {push-ui} in a browser.
+
+. Select the target application from the home page and click *Send Notification To This App*.
+
+. Enter text in the *Message* form.
++
+NOTE: The default priority of a Push Notification is `normal`. To increase the priority, change *Priority* to `high`.
```

Comment:
Now that you added "additional resources" I think we could move this note to its own paragraph.

Reply:
I understand the thinking here, however, I think the reader needs to read this as they move through the procedure as we are not guaranteed that the reader will read the reference material.